Vizzuality/gfw
# Feature/all carbon widget

## Overview

The final climate widget for all carbon (soil and biomass). 

![Screenshot 2019-04-05 at 17 57 48](https://user-images.githubusercontent.com/20288774/55640653-5efb8e80-57cc-11e9-8820-189df4c4a3a1.png)

## Demo

Go to dashboards and checkout and country or admin 1/2

## Testing

- Go to climate tab check it loads
- Change the settings menu
- Change location up and down
